packages/dev/core/src/Events/deviceInputEvents.ts

```diff
@@ -174,6 +174,16 @@ export interface IMouseEvent extends IUIEvent {
      */
     msMovementY?: any;
 
+    /**
+     * Delta of movement on X axis (used with Babylon's internal system, works like movementX)
+     */
+    babylonMovementX?: any;

```

Comment:
Why do you need this? Can we not replace `movementX`?

Reply:
Unfortunately, we can't.  All properties of the `MouseEvents`/`PointerEvents` are readonly so we'd need to create a new object if we wanted overwrite properties like `movementX`.  I figured that morphing the current event is the lesser of two evils.